6-ComandiBase_3/res/slides/11-Footnote.tex

```diff
@@ -0,0 +1,17 @@
+\begin{frame}{Note a piè di pagina}
+
+Per inserire le note a piè di pagina è possibile utilizzare il comando 
+\texttt{footnote}
+
+\vfill
+
+\LaTeX{} si occupa in automatico di posizionamento e della numerazione
+
+\vfill
+
+\begin{exampleblock}{Esempio}
+\dots{}In questo testo c'è una precisazione\textbackslash{}footnote
```

Comment:
Dopo i puntini non ci va la maiuscola (almeno io sapevo così)

Occhio che dopo `footnote` c'è uno spazio, prova a vedere se riesci a toglierlo mettendo un `%` dopo la parola (in teoria funziona perché il compilatore non considera più `/n`.